dockers/docker-fpm-frr/bgpd.conf.default.j2

```diff
@@ -12,7 +12,7 @@ route-map TO_BGP_SPEAKER_V4 deny 10
 {% if prefix | ipv4 and name == 'Loopback0' %}
```

Comment:
there are a few other bgpd.conf template in the docker-fpm-frr folder. Do we need to change those as well?

Reply:
I've redesigned our approach to the config files. Now we have one block defined in one file. Check the changes